### DISCUSSION TITLE
Are project tables available in github enterprise?

### DISCUSSION THREAD

**Discussion by ajungstein** [community/community#38075] [Enterprise]
Doesn't seem to be available in version 3.6. I know the enterprise product tends to lag the public site by about 6 months but was wondering when I can expect this to be available

https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects

**Comment by david-engelmann** [comment]
[Tables for enterprise should be available now, check out this guide](https://docs.github.com/en/enterprise-cloud@latest/issues/planning-and-tracking-with-projects/creating-projects/creating-a-project)

**Reply by ajungstein** [reply]
I didn't seem to see that behaviour when i created a project, but did get it in the public site. Was this recently supported and if so, since which version?
